### DISCUSSION TITLE
BulletList extension

### DISCUSSION THREAD

**Discussion by eyesdwn** [ueberdosis/tiptap#5624] [General Questions]
"Type * , - or + at the beginning of a new line and it will magically transform to a bullet list."

is there a way to deactivate this functionality for '+'? sometimes it breaks the desired flow.

**Comment by hanneswidrig** [comment]
Extending the base extension and overriding certain methods is the easiest way to disable behavior.

```typescript
import BulletListExtension from "@tiptap/extension-bullet-list";

export const BulletList = BulletListExtension.extend({
  addInputRules() {
    return [];
  }
});
```
